docs/installation/linux/centos.md

```diff
@@ -19,7 +19,8 @@ or support Docker on these distributions.
 This page instructs you to install using Docker-managed release packages and
 installation mechanisms. Using these packages ensures you get the latest release
 of Docker. If you wish to install using CentOS-managed packages, consult your
-CentOS documentation.
+CentOS documentation. If you are comfortable with package managers, and are 
```

Comment:
Hm looks like this describes the opposite as what the link refers to; the `curl | sh` is more meant for people that are *not* comfortable with package managers, and not familiar with the prerequisites.